NickvisionMoney.GNOME/org.nickvision.money.metainfo.xml

```diff
@@ -5,10 +5,14 @@
   <project_license>GPL-3.0+</project_license>
   <name>Denaro</name>
   <summary>A personal finance manager</summary>
+  <summary xml:lang="es">Un gestor de finanzas personales</summary>
   <description>
     <p>
       A personal finance manager
     </p>
+    <p xml:lang="es">
+      Un gestor de finanzas personales
+    </p>
```

Comment:
[translate_meta.py](https://github.com/nlogozzo/NickvisionMoney/blob/main/NickvisionMoney.GNOME/translate_meta.py) adds strings here automatically from Strings.*.resx during the build, so these should be removed. Also please remove `<summary>`, I will update the script so it will add it automatically too.